### DISCUSSION TITLE
Hi can you please help me create a page like tik tok or instagram reels?

### DISCUSSION THREAD

**Discussion by marc-walton** [jhomlala/betterplayer#385] [Q&A]
i have two pages under a tabbar with two tabs for videos and live streaming. Im not able to understand the controllers or device orientation as I'm getting an error stating ''The name 'DeviceOrientation' isn't a type so it can't be used as a type argument''.
not just that ,a portrait video is being shrunk to fit the player.
An example code for creating feed type page or tiktok kind will be helpful.
Sorry i don't understand the example codes and also I can't find an example for listvideoplayer.
Please help I'm still learning to code

**Comment by jhomlala** [comment]
Answer is here: https://github.com/jhomlala/betterplayer/issues/386
